backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -133,8 +133,11 @@ namespace /* anonymous */
         buffer_fd = fd;
```

Comment:
Nit: I don't know of any style that indents inside a `namespace`.

Reply:
It is an unfortunate side-effect of the IDE (Android Studio) enforcing its own style. Personally, I do not consider it worth the effort to ensure every contributor has a matching style settings for a minor part of the mainly-Java codebase.